client/src/components/_common/Message/Message.module.scss

```
@@ -61,7 +61,7 @@ p.is-scope-section {
 /* Modifiers: Type */
 
 /* Design decided icon is not necessary for informational messages */
-.is-info .icon {
+.is-info .icon[aria-label='Notice'] {
```

Does the job. And I love attribute selectors.

But aria-label relies on text from markup which, if changed[^1], would be done without second thought to CSS, and thus this fix would be ineffectual. Less likely to change without someone checking for side-effects is a classname.

```suggestion
.is-info .icon:not(.close-icon) {
```

Untested.

[^1]: I admit the change would not happen under our nose (like translation), and is unlikely to happen (we rarely really think much about wording of accessible text), but it's good practice to avoid fragility.